assets/src/web-stories-block/block/components/storiesInspectorControls.js

```diff
@@ -111,6 +128,11 @@ const StoriesInspectorControls = (props) => {
     }
   );
 
+  const archiveLink = `<a target="__blank" href="${archiveURL}">${__(
+    'View archive page',
+    'web-stories'
+  )}</a>`;
```

Comment:
```suggestion
  const archiveLink = (<a target="__blank" href={archiveURL}>{__(
    'View archive page',
    'web-stories'
  )}</a>);
```

Reply:
This results in error, instead need to use following
```suggestion
    const ArchiveLink = () => (
    <a target="__blank" href={archiveURL}>
      {__('View archive page', 'web-stories')}
    </a>
  );
```

This returns a component and gutenberg/react doesn't complain about it.

Found this: https://github.com/facebook/react/issues/13445